jung-visualization/src/main/java/edu/uci/ics/jung/visualization/layout/LayoutTransition.java

```diff
@@ -1,52 +1,68 @@
 package edu.uci.ics.jung.visualization.layout;
 
+import com.google.common.graph.Network;
 import edu.uci.ics.jung.algorithms.layout.Layout;
 import edu.uci.ics.jung.algorithms.layout.StaticLayout;
 import edu.uci.ics.jung.algorithms.layout.util.Relaxer;
 import edu.uci.ics.jung.algorithms.layout.util.VisRunner;
 import edu.uci.ics.jung.algorithms.util.IterativeContext;
 import edu.uci.ics.jung.visualization.VisualizationViewer;
+import edu.uci.ics.jung.visualization.util.LayoutMediator;
 import java.awt.geom.Point2D;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class LayoutTransition<V, E> implements IterativeContext {
```

Comment:
The start and end graph do not need to be the same. ShowLayouts changes Graphs the same way.  Did you run it?

Is the last paragraph a trick question? :)
It uses the LayoutMediator because Layout does not have a getGraph.